### PR TITLE
fix(log-to-customer): fix duplicate tracking on subscribe

### PR DESCRIPTION
**Pull request by rjamet-ovh**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #MANAGER-15763
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description

fix duplicate tracking tag on click on "subscribe to another data-stream" button

### CODE REVIEW

File: packages/manager/modules/log-to-customer/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ovh-ux/manager-log-to-customer",
-  "version": "2.3.1",
+  "version": "2.3.2",
```
Comment (tibs245) [inline packages/manager/modules/log-to-customer/package.json:3]: It's not a automatic edition by cds ?

**Review by tibs245 — APPROVED** [review]

File: packages/manager/modules/log-to-customer/package.json (hunk 1 above)
Comment (darsene) [inline packages/manager/modules/log-to-customer/package.json:3]: Why did you change the version ?

**Review by tibs245 — APPROVED** [review]

**Review by darsene — APPROVED** [review]
